homeassistant/components/freedompro/config_flow.py

```diff
@@ -0,0 +1,75 @@
+"""Config flow to configure Freedompro."""
+from pyfreedompro import get_list
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+from homeassistant.helpers import aiohttp_client
+
+from .const import DOMAIN  # pylint:disable=unused-import
+
+STEP_USER_DATA_SCHEMA = vol.Schema({"api_key": str})
```

Comment:
Use [CONF_API_KEY](https://github.com/home-assistant/core/blob/cb1b45d31ab85edec72caf3fee77bba3c9c6e425/homeassistant/const.py#L36) instead of "api_key". And import: `from homeassistant.const import CONF_API_KEY`.
Make api_key required, [L31](https://github.com/home-assistant/core/pull/46332/files#diff-7c754d47535aea42c0a53f79ef22fc5b53992f53d5fd647d90f9f0ec8d46ef32R31) will fail if no api_key is given. `vol.Required()`

```suggestion
STEP_USER_DATA_SCHEMA = vol.Schema({vol.Required(CONF_API_KEY): str})
```